lib/mobility/backends/active_record/container.rb

```diff
@@ -37,7 +37,7 @@ def read(locale, _ = nil)
       # @return [String,Integer,Boolean] Updated value
       def write(locale, value, _ = nil)
         set_attribute_translation(locale, value)
-        read(locale)
+        value
```

Comment:
This will sometimes return the wrong value, e.g. if you have the presence plugin enabled and assign `""`, this would return `""` whereas it should return `nil`.